handbook/technical-overview/auth/system/service-provider-manual/authentication-service-provision.md

```diff
@@ -0,0 +1,59 @@
+
+Further information regarding the development- and service provider-environments can be found following these links:
+
+- [https://docs.devland.is/handbook/technical-overview/devops](https://docs.devland.is/handbook/technical-overview/devops)
+- [https://docs.devland.is/handbook/projects/services/auth-api](https://docs.devland.is/handbook/projects/services/auth-api)
+- [https://docs.devland.is/handbook/projects/services/auth-admin-api](https://docs.devland.is/handbook/projects/services/auth-admin-api)
+- [https://github.com/island-is/identity-server.web](https://github.com/island-is/identity-server.web)
+
+## IdentityServer
+
+**[IdentityServer](https://identityserver4.readthedocs.io/en/latest/)** is an open source-solution in **[.NetCore](https://en.wikipedia.org/wiki/.NET_Core)**. The system is an **[OpenID Connect provider](https://openid.net/connect/)**, i.e. it implements the **OpenID Connect** and **OAuth 2.0** protocols.
+
+The system is divided into three components:
+
+### User
+
+A human that is using a registered client to access resources.
+
+### Client
+
+A software that requests tokens from **IdentityServer** - either for authenticating a user or for accessing a resource. A client must be first registered with **IdentityServer** before it can request tokens. Clients can for example be web applications, native mobile or desktop applications.
+
+### Resources
+
+Resources are something you want to protect with **IdentityServer** - either identity data of your users, or APIs:
+
+- Identity data to identify users, for example names or e-mail addresses.
+- API is an interface which the clients communicate with.
+
+### Identity token
+
+An identity token is created during the authentication procedure of **IdentityServer**. It contains at a bare minimum an identifier for the user (called the _sub_ aka _subject claim_) and information about how and when the user authenticated.
+
+### Access token
+
+An access token allows access to an API resource. Clients request access tokens and forward them to the API. Access tokens contain information about the client and the user which the API uses to authorize access to the resources.
+
+### Authentication flow
+
+1. \***\*The service provider writes a code to connect with **[Identity server](https://identityserver.io/)\**‘s authentication service. The user’s identity is defined as a *sub* (*subject claim\*) in the code.
+2. \***\*When the users’ clients request authentication then **IdentityServer** receives the _sub_. Accordingly, **IdentityServer** includes the _sub_ in the identity token which it creates to connect the clients with the resources. During the same procedure **IdentityServer\*\* also creates the access token.
+3. **IdentityServer** sends the identity token and the access token to the client.
+4. \*\*\*\*The client forwards the access token to the API for validation.
+5. \*\*\*\*When the API has validated the access token then the client gets access to the user’s resources.
```

Comment:
Why the different use for emphasis, sometimes bold, sometimes italic and sometimes starred?
![image](https://user-images.githubusercontent.com/54210288/103887485-052fa300-50db-11eb-85df-96ed46371a88.png)

Reply:
Not sure if you're asking why it's sometimes bold or not :)... but here it goes:
one star * is italic: *I'm italic*
two stars ** is bold: **I'm bold**
escaping star \ ignores formatting

My question was rather, why having \*\*\*\* after each no, and why not use backslashes in some places `prod`?